src/components/Header.tsx

```diff
@@ -0,0 +1,54 @@
+export default function Header() {
+  return (
+    <header className="flex justify-between items-center m-2">
+      <img src="header-logo.png" alt="logo" className="h-6 w-6" />
+      <div className="flex items-center gap-2">
+        <button className="p-2 rounded-full" style={{ backgroundColor: "#1f1f1f" }}>
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            height="24px"
+            viewBox="0 -960 960 960"
+            width="24px"
+            fill="#e8eaed"
+          >
+            <path d="M160-120v-480l320-240 320 240v480H560v-280H400v280H160Z" />
+          </svg>
```

Comment:
for later , we can move this to icons directory in components folder

Reply:
Okay sure we can.